fhir-path/src/test/java/com/ibm/fhir/path/test/FHIRPathAsTest.java

```diff
@@ -47,29 +47,29 @@ void testAsFunction() throws Exception {
 
         FHIRPathEvaluator evaluator = FHIRPathEvaluator.evaluator();
         Collection<FHIRPathNode> result = evaluator.evaluate(condition, "Condition.onset.as(Age) | Condition.onset.as(Range)");
-        
+
         assertEquals(result.size(), 0, "Number of selected nodes");
     }
-    
+
     @Test
     void testResolveAsOperation() throws Exception {
         Patient patient = Patient.builder()
-                                 .generalPractitioner(Reference.builder().reference(string("http://example.com/dummyReference")).build())
+                                 .generalPractitioner(Reference.builder().reference(string("http://example.com/Practitioner/1234")).build())
```

Comment:
@JohnTimm was this test intentionally testing an absolute reference that didn't follow the FHIR rest API pattern?
@tbieste if so, we might want to revert this particular edit.

Reply:
From the other comment. Validation of the resource type is skipped on references starting with "http:" or "https:", so yes, this can be reverted.

Reply:
Reverted.